component/client/chat/ClientChat.tsx

```diff
@@ -34,27 +33,25 @@ class ClientChat extends React.Component<IClientChatProps, IClientChatState> {
   }
 
   public componentWillUnmount(): void {
-    if (this.roomService) this.roomService.close();
+    room.close();
   }
 
   public render(): JSX.Element {
     console.log("[ INFO ] : ClientChat render url and messages", this.props.url, this.state.messages);
     return (
       <div>
-        <Panel header="Chat" bsStyle="primary" footer={this.messenger}>
+        <Panel header={"Chat at " + this.props.url} bsStyle="primary" footer={this.messenger}>
           <ChatHistory messages={this.state.messages} />
         </Panel>
       </div>
     );
   }
 
   private updateRoomService(url: string) {
-    if (this.roomService)
-      this.roomService.close();
     this.state = {
       messages : []
     };
-    this.roomService = new RoomService(url, (data: IData, user) => {
+    room.setUrl(url, (data: IData, user) => {
```

Comment:
updateConf() and setConf() allows you to dynamically add user configuration.
I did not constraint it because that makes it flexible.
You can configure your group aliases or any other data, so when you send a data other people will get the data you configured.